float32 joystick_left_y 
float32 joystick_left_x 
float32 joystick_right_y
float32 joystick_right_x 
float32 trigger_right
float32 trigger_left


uint8 bumper_left
uint8 bumper_right
uint8 button_x 
uint8 button_y 
uint8 button_a 
uint8 button_b